pkg/packetfilter/nftables/nftables.go

```diff
@@ -334,7 +334,9 @@ func ToRuleSpec(rule *packetfilter.Rule) RuleSpec {
 	}
 
 	if rule.MarkValue != "" && rule.Action != packetfilter.RuleActionMark {
-		ruleSpec = append(ruleSpec, "mark", rule.MarkValue)
+		// syntax for MarkValue '0xc0000/0xc0000': meta mark & 0xc0000 == 0xc0000
+		i := strings.Index(rule.MarkValue, "/")
+		ruleSpec = append(ruleSpec, "meta", "mark", "&", rule.MarkValue[:i], "==", rule.MarkValue[:i])
```

Comment:
What does the syntax "0xc0000/0xc0000" mean? Should we just assume this implicit syntax of 2 terms separated by a '/'? Since we're only using the first term before the '/' and discarding the second here, is this syntax specific to the iptables implementation? Or should this be:

```suggestion
		ruleSpec = append(ruleSpec, "meta", "mark", "&", rule.MarkValue[:i], "==", rule.MarkValue[i+1:])
```

PS: it might be useful to include an explanation of the intent/changes in the commit/PR description.

Reply:
Yep, good point.
I updated the commit and PR description.